iOS-27 | Добавил кэширование фотографий в FileManager

mightyK1ngRichard opened the pull request.

mightyK1ngRichard committed:
iOS-27: Добавлено кэширование фоток. Обновление моделей

mightyK1ngRichard committed:
iOS-27: Удалил тестовый экран

itissep commented on CakesHub/CakesHub/Services/Firebase/Entities/FBProductModel/Substructures/FBCommentInfoModel.swift:
```diff
@@ -16,15 +16,13 @@ extension FBProductModel {
             description: .clear,
-            countFillStars: 0,
-            feedbackCount: 0
+            countFillStars: 0
```
не очень нравится это название для переменной. это же количество звезд на ревью?

mightyK1ngRichard replied:
боюсь, поменять уже будет очень тяжко

itissep commented on CakesHub/CakesHub/Services/Firebase/Entities/FBProductModel/Substructures/FBProductReviewsModel.swift:
```diff
@@ -44,7 +46,8 @@ extension FBProductModel.FBProductReviewsModel {
             countTwoStars: dictionary["countTwoStars"] as? Int ?? 0,
             countOneStars: dictionary["countOneStars"] as? Int ?? 0,
             countOfComments: dictionary["countOfComments"] as? Int ?? 0,
-            comments: comments.compactMap { .init(dictionary: $0) }
+            comments: comments.compactMap { .init(dictionary: $0) },
+            feedbackCount: dictionary["feedbackCount"] as? Int ?? 0
```
feedbackCount про звезды?

mightyK1ngRichard replied:
Да, сколько раз ставили звёздочки

mightyK1ngRichard committed:
iOS-28: Создал экран создания отзыва

mightyK1ngRichard committed:
iOS-28: Добавил логику для экрана отзыва

mightyK1ngRichard committed:
iOS-28: Доработка экрана отзыва и моделей

1. Добавил обновление информации о товаре при оставке отзыва
2. Поправил расчёты логики рейтинга товара

mightyK1ngRichard committed:
iOS-28: Убрал анимацию при обновлении категорий

mightyK1ngRichard committed:
iOS-29: Добавил локальные пуши с задержкой

mightyK1ngRichard committed:
iOS-30: Обновил шаблон

mightyK1ngRichard committed:
iOS-30: Добавлена карта

mightyK1ngRichard committed:
iOS-30: minor

mightyK1ngRichard committed:
iOS-30: Minor

mightyK1ngRichard committed:
iOS-30: fix bug

mightyK1ngRichard committed:
iOS-31: Добавил архитектуру для экрана насроек и логику выхода

mightyK1ngRichard committed:
iOS-31: Доработка AuthView под новый подход

mightyK1ngRichard committed:
iOS-31: minor

mightyK1ngRichard committed:
Merge pull request #43 from iOS-vk-education/feature/permyakov/iOS-31_logout

iOS-31 | Добавлена архитектура для экрана настроек. Внедрена логика выхода их аккаунта

mightyK1ngRichard committed:
Merge pull request #42 from iOS-vk-education/feature/permyakov/iOS-30_map_view

iOS-30 | Добавлена карта

mightyK1ngRichard committed:
Merge pull request #41 from iOS-vk-education/feature/permyakov/iOS-29_add_local_push_notifications

iOS-29 | Добавлены локальные пуши

mightyK1ngRichard committed:
Merge pull request #40 from iOS-vk-education/feature/permyakov/iOS-28_add_feedback_screen

iOS-28 | Добавлен экран фидбэка о товаре
